crates/bevy_pbr/src/render/light.rs

```diff
@@ -92,9 +92,87 @@ pub struct GpuPointLight {
     shadow_normal_bias: f32,
 }
 
-#[derive(AsStd140)]
-pub struct GpuPointLights {
-    data: [GpuPointLight; MAX_POINT_LIGHTS],
+pub enum GpuPointLights {
```

Comment:
I agree. I'm just not sure what it is yet. In some cases we may want to store stuff in textures instead of uniform buffers and then do textureLoad and try to reconstruct the data.